src/sine.c

```diff
@@ -497,9 +500,10 @@ float est_voicing_mbe(
 
         offset = FFT_ENC/2 - l*Wo*FFT_ENC/TWO_PI + 0.5;
 	for(m=al; m<bl; m++) {
-	    Am.real += Sw[m].real*W[offset+m].real;
-	    Am.imag += Sw[m].imag*W[offset+m].real;
-	    den += W[offset+m].real*W[offset+m].real;
+	    Am.real += Sw[m].real*W[offset+m];
+	    Am.imag += Sw[m].imag*W[offset+m];
+
+	    den += W[offset+m]*W[offset+m];
```

Comment:
Good catch - might be useful as a general mod (x86 and embedded) if W is always real.